bin/install

```diff
@@ -6,43 +6,37 @@
 # Uses the latest release, unless a version identifier is specified as a parameter of this script.
 
 command_exists() {
-	command -v "$@" > /dev/null 2>&1
+    command -v "$@" > /dev/null 2>&1
 }
 
 install_version() {
-    echo "Using $1 to install minimesos $2"
-    curl -sSL $1 | bash -s $2
+    curl -sSL $1 | sh -s $2
 }
 
 if ! command_exists curl; then
-	echo "Please install curl to fetch the minimesos files"
-	exit 1
+    echo "Please install curl to fetch the minimesos files"
+    exit 1
 fi
 
 VERSION=$(echo $@ | xargs)
 
 if [ -z "$VERSION" ]; then
-	MAVEN_METADATA=$(curl -s https://jitpack.io/com/github/ContainerSolutions/minimesos/maven-metadata.xml)
-	VERSION=$(echo "$MAVEN_METADATA" | awk -F '[<>]' '/release/{print $3}')
-	if [ ! "$VERSION" ]; then
-		echo "Cannot determine latest release of minimesos"
-		exit 1
-	fi
+    MAVEN_METADATA=$(curl -s https://jitpack.io/com/github/ContainerSolutions/minimesos/maven-metadata.xml)
```

Comment:
Maybe in the next release, which is soon. Parsing JSON, not assuming to find the utility on the path, is tricky